modules/library/referencing/src/main/java/org/geotools/measure/Units.java

```diff
@@ -130,8 +132,7 @@ public static final boolean isDegreeAngle(Unit<?> unit) {
             if (converter instanceof MultiplyConverter) {
                 MultiplyConverter multiplyConverter = (MultiplyConverter) converter;
                 double factor = multiplyConverter.getFactor();
-                if (factor == 0.017453292519943295 || factor == 0.0174532925199433
-                        || factor == 0.01745329251994328) {
+                if (Math.abs(RADIAN_TO_DEGREE_RATIO - factor) < 1.0e-17) {
```

Comment:
@jodygarnett I would prefer a tolerance as a static final constant. This also looks too small too me. Subtractive cancellation can easily lose several orders of magnitude of precision. If I compare the three constants removed with the tolerance, you can already see that the last will already differ by more than the the tolerance:
```
0.017453292519943295
0.017453292519943300
0.017453292519943280
0.000000000000000010
```
How about 1e-15?

Reply:
Thanks Ben, relaxing this tolerance has referencing passing (especially now that @dispiste uses the test while parsing)